data/simple-icons.json

```diff
@@ -11382,6 +11382,20 @@
 		"source": "https://en.facebookbrand.com/oculus/assets/oculus?audience=oculus-landing",
 		"guidelines": "https://en.facebookbrand.com/oculus/"
 	},
+	{
+		"title": "Odido",
+		"hex": "000000",
+		"source": "https://www.odido.nl/Assets/Lfwd/Static/odido-logo-black.svg",
+		"guidelines": "https://merk.odido.nl/en/logo",
+		"aliases": {
+			"aka": [
+				"ODIDO"
```

Comment:
This alternative casing alias is unnecessary. And I also don't see them referring to themselves this way anywhere.